utils/scm/applications/mysql/mysql_data_persistence.yml

```diff
@@ -18,9 +18,49 @@
 
 - block:
 
+   - name: Kill the application pod
+     shell: >
+       kubectl delete pod {{ pod_name }} -n {{ ns }}
+     args:
+       executable: /bin/bash
+
+   - name: Verify if the applicaion pod is deleted
+     shell: >
+       kubectl get pods -n {{ ns }}
+     args:
+       executable: /bin/bash
+     register: podstatus
+     until: '"{{ pod_name }}" not in podstatus.stdout'
+     retries: 2
+     delay: 150
+
+   - name: Obtain the newly created pod name for applicaion
+     shell: >
+       kubectl get pods -n {{ ns }} -o jsonpath='{.items[].metadata.name}'
```

Comment:
lets get the label of the app. How do we deal with cases where there are more app replicas?